tests/system_library/BUILD

```diff
@@ -7,4 +7,7 @@ sh_test(
         "//cc:system_library.bzl",
         "@bazel_tools//tools/bash/runfiles",
     ],
+    target_compatible_with = [
```

Comment:
Is this change related?

Reply:
The relation is that I wanted to include a test, but noticed that `//tests/...` isn't run in CI. I created a separate PR with just this change: https://github.com/bazelbuild/rules_cc/pull/188